addons/mail/static/src/models/timer.js

```diff
@@ -1,162 +1,123 @@
 /** @odoo-module **/
 
-import { registerModel } from '@mail/model/model_core';
-import { attr, one } from '@mail/model/model_field';
-import { clear } from '@mail/model/model_field_command';
+import { addFields, patchFields, patchRecordMethods } from '@mail/model/model_core';
+import { one } from '@mail/model/model_field';
+// ensure that the model definition is loaded before the patch
+import '@mail/core_models/timer';
 
-registerModel({
-    name: 'Timer',
-    identifyingMode: 'xor',
-    lifecycleHooks: {
-        _created() {
-            this.update({ timeoutId: this.messaging.browser.setTimeout(this._onTimeout, this.duration) });
-        },
-        _willDelete() {
-            this.messaging.browser.clearTimeout(this.timeoutId);
-        },
-    },
-    recordMethods: {
-        /**
-         * @private
-         */
-        _onTimeout() {
-            this.update({ timeoutId: clear() });
+addFields('Timer', {
```

Comment:
the diff in this file is really not great, we should wait for the PR changing patch syntax if it's almost ready